config/flags_service_openvpn.go

```diff
@@ -46,6 +46,18 @@ var (
 		Usage: "OpenVPN subnet netmask",
 		Value: "255.255.255.0",
 	}
+	// FlagOpenVPNPriceMinute sets the price per minute for provided OpenVPN service.
+	FlagOpenVPNPriceMinute = cli.Uint64Flag{
+		Name:  "openvpn.price-minute",
+		Usage: "Sets the price of the OpenVPN service per minute.",
+		Value: 50000,
```

Comment:
Human unreadable. IMHO user should input MYST value: 0.0006
@chompomonim, opinions?

Reply:
Isn't 0.0006 more confusing and hard to manipulate?

Reply:
User does not know decimal size of our token, why should he bother finding it, understanding how much real MYST is it

Reply:
Changed it to display in MYST.